nifi-manifest/nifi-runtime-manifest-test/src/test/java/org/apache/nifi/runtime/manifest/TestRuntimeManifest.java

```diff
@@ -139,6 +142,20 @@ public void testRuntimeManifest() throws IOException {
         assertEquals(1, resourceDefinition.getResourceTypes().size());
         assertEquals(ResourceType.FILE, resourceDefinition.getResourceTypes().stream().findFirst().get());
 
+        // Verify FetchHDFS definition has restrictions
```

Comment:
What do you think about adding assertion for a negative case too? I mean where the ExtensionComponnet is not set to be restricted.

Reply:
Good call, I meant to do that, pushed a new commit